pkg/controller/process.go

```diff
@@ -46,6 +45,13 @@ func (c AppGwIngressController) Process(event events.Event) error {
 		EnableBrownfieldDeployment: envVars.EnableBrownfieldDeployment == "true" && len(prohibitedTargets) > 0,
 	}
 
+	cbCtx.IngressList = c.PruneIngress(&appGw, cbCtx)
+	if len(cbCtx.IngressList) == 0 {
+		errorLine := "no Ingress in the pruned Ingress list. Please check Ingress events to get more information"
+		glog.Error(errorLine)
+		return errors.New(errorLine)
```

Comment:
I hope one day we get a chance to move all our errors in a central location somewhere...